app/assets/javascripts/training/components/training_library_handler.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import CreateLibrary from './modals/create_library';
+import Notifications from '../../components/common/notifications.jsx';
+import { getCurrentUser } from '../../selectors';
+
+const TrainingLibraryHandler = () => {
+  const [showModal, setShowModal] = useState(false);
+  const currentUser = useSelector(state => getCurrentUser(state));
+
+  const modalHandler = () => {
+    setShowModal(!showModal);
+  };
```

Comment:
`toggleModal` would be a better name here I think

Reply:
sure